src/aat/AATFeatureMap.js

```diff
@@ -478,14 +478,14 @@ for (let ot in OTMapping) {
 // in the form of {featureType:{featureSetting:true}}
 export function mapOTToAAT(features) {
   let res = {};
-  for (let k = 0; k < features.length; k++) {
+  for (let k in features) {
```

Comment:
`in` is a pretty dangerous old JS construction, are you sure you don't want `features.forEach( (feature, k) => { ... })`; here?